local.yml

```diff
@@ -0,0 +1,47 @@
+
+---
+version: "3"
+
+volumes:
+  traefik_acme_data: {}
+  local_postgres_data: {}
+
+services:
+  traefik:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/traefik/Dockerfile
+    image: mlflow_production_traefik
+    restart: always
+    volumes:
+      - traefik_acme_data:/etc/traefik/acme/
+      - ./config/.htpasswd:/etc/traefik/.htpasswd:ro
+      - ./config/traefik.yml:/etc/traefik/traefik.yml:ro
+    ports:
+      - "0.0.0.0:80:8080"
+      - "0.0.0.0:443:8443"
+
+
+  mlflow:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/mlflow/Dockerfile
+    image: mlflow_production_server
+    restart: always
+    command: /start
+    ports:
+      - "5000:5000"
+    depends_on:
+      - postgres
+    env_file:
+      - .env
+
+  postgres:
+    image: postgres:11.8
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - local_postgres_data:/var/lib/postgresql/data
```

Comment:
this is the only change over the other file, just drop in a new docker-compose-local.yml file and can run docker-compose up with both rather than copy paste